tracking_manager/tests/test_tracking_manager.py

```diff
@@ -163,7 +160,7 @@ def test_o2m_write_indirectly_on_not_tracked_fields(self):
                 "bank_ids": [(1, self.partner.bank_ids[0].id, {"acc_number": "123"})],
             }
         )
-        self.assertEqual(len(self.messages), 0)
+        self.assertEqual(len(self.messages), 1)
```

Comment:
Why 1 instead of 0?

Reply:
I tried with a clean installation of odoo and there Is already tracking enabled on those one2many.
If i have not this module installed, a note Is written with the changed value  log